test/unit/webistrano_deployer_test.rb

```diff
@@ -956,6 +888,15 @@ def assert_correct_task_called(task_name)
     # get things started
     deployer = Webistrano::Deployer.new(@deployment)
     deployer.invoke_task!
+    remove_directory(@deploymen.stage.project.webistrano_project_name)
+  end
+
+  def remove_directory(project)
+    FileUtils.rm_rf(rooted("capsize_projects/#{project}"))
   end
-  
+
+  def rooted(dir)
+    Rails.root.join(dir)
+  end
```

Comment:
This is duplicated code. You should probably put it in a module that can be included. You can also provide other kinds of file accessors in that module, so that there aren't as many constants littering your codebase.